contracts/gauge/Gauge.sol

```diff
@@ -0,0 +1,217 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7; 
+
+import "../libraries/SafeMath.sol";
+import "../libraries/SafeERC20.sol";
+import "../libraries/ProtocolGovernance.sol";
+import "../libraries/Math.sol";
+import "../libraries/reentrancy-guard.sol";
+
+
+contract Gauge is ProtocolGovernance, ReentrancyGuard {
```

Comment:
let continue the convention of `GaugeVn`

Reply:
@bayological When you get a moment could you explain to me what this comment means?